libraries/bdp-access-proxy/bdp-access-impl/src/main/java/com/bosch/osmi/bdp/access/impl/model/LicenseImpl.java

```diff
@@ -60,7 +60,7 @@ public boolean hasLicenseTextAttached(){
         } catch (SdkFault sdkFault) {
             LOGGER.error("Unable to get license text from Bdp server. Reason: \n" + sdkFault.getMessage());
             LOGGER.debug("License ID: " + getId());
-            throw new IllegalStateException(sdkFault.getCause());
+            return false;
```

Comment:
We scanned it once more: for documents and lists of documents failure is reported as an exception. For low level attributes of the documents, default values are returned.